hw5/hw5.py

```diff
@@ -0,0 +1,227 @@
+
+
+import pandas as pd
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+import math
+import matplotlib.font_manager as font_manager
+import matplotlib.ticker as ticker
+from matplotlib.patches import ConnectionPatch
+
+
+def read_bed6(path, headers = ["chromosome","start", "end", "name", "score", "strand"]):
+    alignment = pd.read_table(path, header=None, names = headers)
+    return alignment
+
+alignment = read_bed6("/content/alignment.bed")
+alignment
+
+
+def read_gff(path, headers = ["chromosome", "source", "type", "start", "end", "score", "strand", "phase", "attributes"]):
+    annotation = pd.read_table(path, header=0, names = headers)
```

Comment:
Твой код учитывает только случай, где комментарий занимает только первую строку. Однако чаще всего в файлах данного формата могут быть комментарии на много строк, их нужно учитывать. Благо в пандасе это делается супер просто
```suggestion
    annotation = pd.read_table(path, comment="#", names=headers)
```